modules/apps/commerce/commerce-service/src/main/java/com/liferay/commerce/model/impl/CommerceAddressImpl.java

```diff
@@ -63,6 +66,16 @@ public static CommerceAddress fromAddress(Address address) {
 			}
 		}
 
+		List<Phone> phones = PhoneLocalServiceUtil.getPhones(
+			address.getCompanyId(), Address.class.getName(),
+			address.getAddressId());
+
+		if (!phones.isEmpty()) {
+			Phone phone = phones.get(0);
```

Comment:
why get 0?

Reply:
Hey @christian-chiappa I had a look on other model impl I believe you cannot use the finders here, so this change is fine.